ios/Classes/AblyFlutterStreamHandler.m

```diff
@@ -82,32 +82,32 @@ - (void) cancelListening:(AblyFlutterMessage *const)message {
     NSString *const eventName = eventMessage.eventName;
     
     if([AblyPlatformMethod_onRealtimeConnectionStateChanged isEqual: eventName]) {
-        [[_ably realtimeFrom:handle].connection off: listener];
+        [[_instanceStore realtimeFrom:handle].connection off: listener];
     } else if([AblyPlatformMethod_onRealtimeConnectionStateChanged isEqual: eventName]) {
         // Note: this and all other assert statements in this onCancel method are
         // left as is as there is no way of propagating this error to flutter side
         NSAssert(eventMessage.message, @"event message is missing");
         NSMutableDictionary<NSString *, NSObject *>* eventPayload = eventMessage.message;
-        ARTRealtime* realtime = [_ably realtimeFrom:handle];
+        ARTRealtime *const realtime = [_instanceStore realtimeFrom:handle];
         
-        NSString *channelName = (NSString*)[eventPayload objectForKey:TxTransportKeys_channelName];
-        ARTRealtimeChannel *channel = [realtime.channels get:channelName];
+        NSString *const channelName = (NSString*) eventPayload[TxTransportKeys_channelName];
+        ARTRealtimeChannel *const channel  = [realtime.channels get:channelName];
         [channel off: listener];
     } else if([AblyPlatformMethod_onRealtimeConnectionStateChanged isEqual: eventName]) {
         NSAssert(eventMessage.message, @"event message is missing");
         NSMutableDictionary<NSString *, NSObject *>* eventPayload = eventMessage.message;
-        ARTRealtime* realtime = [_ably realtimeFrom:handle];
+        ARTRealtime *const realtime = [_instanceStore realtimeFrom:handle];
         
-        NSString *channelName = (NSString*)[eventPayload objectForKey:TxTransportKeys_channelName];
-        ARTRealtimeChannel *channel = [realtime.channels get:channelName];
+        NSString *const channelName = (NSString*) eventPayload[TxTransportKeys_channelName];
+        ARTRealtimeChannel *const channel  = [realtime.channels get:channelName];
         [channel unsubscribe: listener];
     } else if ([AblyPlatformMethod_onRealtimePresenceMessage isEqual: eventName]) {
         NSAssert(eventMessage.message, @"event message is missing");
         NSMutableDictionary<NSString *, NSObject *>* eventPayload = eventMessage.message;
-        ARTRealtime* realtime = [_ably realtimeFrom:handle];
+        ARTRealtime *const realtime = [_instanceStore realtimeFrom:handle];
         
-        NSString *channelName = (NSString*)[eventPayload objectForKey:TxTransportKeys_channelName];
-        ARTRealtimeChannel *channel = [realtime.channels get:channelName];
+        NSString *const channelName = (NSString*) eventPayload[TxTransportKeys_channelName];
```

Comment:
```suggestion
        NSString *const channelName = eventPayload[TxTransportKeys_channelName];
```

Reply:
I'm getting linter warning in AppCode here if I do this though:

![CleanShot 2021-11-29 at 09 37 02](https://user-images.githubusercontent.com/24711048/143843359-e9d19bd4-4a63-4037-8423-8f9026d1e121.png)

Reply:
And on Xcode:

![CleanShot 2021-11-29 at 09 38 46](https://user-images.githubusercontent.com/24711048/143843657-e9af7555-dc30-40fe-82c0-9fdfc4e24f8b.png)